package.json

```diff
@@ -23,7 +25,7 @@
     "jsdoc-to-markdown": "^3.x",
     "postcss-js": "^0.x",
     "standard": "^10.x",
-    "standard-version": "^4.0.0",
+    "standard-version": "^4.x",
     "sugarss": "^1.x",
     "webpack": "^2.x"
```

Comment:
These version numbers look wrong. You're already using the semver caret; using the `x` placeholder probably only works by accident.